cli/internal/run/dry_run.go

```diff
@@ -215,6 +240,23 @@ func displayDryTextRun(ui cli.Ui, summary *dryRunSummary, workspaceInfos graph.W
 		}
 	}
 
+	fileCount := 0
+	for range summary.GlobalHashSummary.GlobalFileHashMap {
+		fileCount = fileCount + 1
+	}
+	w1 := tabwriter.NewWriter(os.Stdout, 0, 0, 1, ' ', 0)
+	ui.Output("")
+	ui.Info(util.Sprintf("${CYAN}${BOLD}Global Hash Inputs${RESET}"))
+	fmt.Fprintln(w1, util.Sprintf("  ${GREY}Global Files\t=\t%d${RESET}", fileCount))
+	fmt.Fprintln(w1, util.Sprintf("  ${GREY}External Dependencies Hash\t=\t%s${RESET}", summary.GlobalHashSummary.RootExternalDepsHash))
+	fmt.Fprintln(w1, util.Sprintf("  ${GREY}Global Cache Key\t=\t%s${RESET}", summary.GlobalHashSummary.GlobalCacheKey))
+	if bytes, err := json.Marshal(summary.GlobalHashSummary.Pipeline); err == nil {
+		fmt.Fprintln(w1, util.Sprintf("  ${GREY}Root pipeline\t=\t%s${RESET}", bytes))
+	}
+	if err := w1.Flush(); err != nil {
+		return err
+	}
```

Comment:
example of what this looks like:
<img width="1961" alt="CleanShot 2023-02-23 at 11 50 33@2x" src="https://user-images.githubusercontent.com/490968/221014816-2ed6a97c-f827-44fc-91b2-afc78d8e86be.png">

Reply:
I would dim these key / vals as well, but not a blocker.

Reply:
I did that in a previous pass, but then felt it was more like the Packages In scope section rather than tasks section. I'll make a note of this and come back to it once all the pieces are in